week-3/C-array-every/exercise.js

```diff
@@ -5,7 +5,15 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+function Check(names) {
```

Comment:
This is not quite right. For example, if group was `["Austine", "Dany"]`, `group.every(Check)` should return true.
What are the inputs and outputs to the Check function? Can you explain in words?